app/renderer/js/pages/preference/preference.ts

```diff
@@ -1,24 +1,28 @@
 'use strict';
 
-const BaseComponent = require(__dirname + '/js/components/base.js');
-const { ipcRenderer } = require('electron');
+import { ipcRenderer } from 'electron';
 
-const Nav = require(__dirname + '/js/pages/preference/nav.js');
-const ServersSection = require(__dirname + '/js/pages/preference/servers-section.js');
-const GeneralSection = require(__dirname + '/js/pages/preference/general-section.js');
-const NetworkSection = require(__dirname + '/js/pages/preference/network-section.js');
-const ConnectedOrgSection = require(__dirname + '/js/pages/preference/connected-org-section.js');
-const ShortcutsSection = require(__dirname + '/js/pages/preference/shortcuts-section.js');
+import BaseComponent = require('./js/components/base');
+import Nav = require('./js/pages/preference/nav');
+import ServersSection = require('./js/pages/preference/servers-section');
+import GeneralSection = require('./js/pages/preference/general-section');
+import NetworkSection = require('./js/pages/preference/network-section');
+import ConnectedOrgSection = require('./js/pages/preference/connected-org-section');
+import ShortcutsSection = require('./js/pages/preference/shortcuts-section');
 
 class PreferenceView extends BaseComponent {
+	$sidebarContainer: Element;
+	$settingsContainer: Element;
+	nav: typeof Nav;
+	section: typeof ServersSection | typeof GeneralSection |
+		typeof NetworkSection | typeof ConnectedOrgSection | typeof ShortcutsSection;
```

Comment:
Let extract all of this out to a variable, i.e. `type Section = typeof ...` and use it there.